helm-chart/images/jupyterhub-ssh/jupyterhub_ssh_config.py

```diff
@@ -4,6 +4,9 @@
 with open("/etc/jupyterhub-ssh/config/values.yaml") as f:
     config = yaml.load(f)
 
-c.JupyterHubSSH.host_key_path = "/etc/jupyterhub-ssh/config/hostKey"
+# FIXME: help this config migrate to ssh.config as well
 c.JupyterHubSSH.hub_url = config["hubUrl"]
```

Comment:
Let's remove this special case too? I think the breaking change is ok. Will helm still be able to validate them?

Reply:
Okay hmmm, if this is configured via `ssh.config.hub_url`, then where should the `sftp` configuration go? Should it remain at the root level or should it be at `sftp.hubUrl`, or something else?

Reply:
I'd like to see these charts be split apart in the future, so I like to migrate the `hub_url` config for the ssh part of the chart to `ssh.config.hub_url` but it is a bit complicated until we have a proper split between the charts.

Perhaps we can wait with this? I opted out of doing it now feeling that it added significant complexity to the PR that could be tackled later while making a split of the charts as I'd like to do.

Reply:
Btw if someone does use `hub_url` in `ssh.config`, it will be valid and functional as well and take precidence over what was configured in `hubUrl`.